rbc/tests/test_omnisci_text_encoding.py

```diff
@@ -14,23 +14,25 @@ def omnisci():
 
 
 def define(omnisci):
-    @omnisci("int32(Column<T>, RowMultiplier, OutputColumn<T>)",
+    @omnisci("int32(Column<T>, RowMultiplier, OutputColumn<T> | input_id=args<0>)",
```

Comment:
Notice:
- omniscidb (https://github.com/omnisci/omniscidb-internal/pull/5719) uses format `OutputColumn<T:label1=annotation1:label2=annotation2>`.
- rbc uses format `OutputColumn<T> | label1=annotation1 | label2=annotation2`

that is, omniscidb annotations are attached to item types while rbc annotations are attached to argument types.
So, we have two different formats for the same thing.

Are there any reasons why omniscidb annotations should
1. not be attached to argument types? 
2. not use `|` to separate annotations?

Reply:
One advantage of having annotations attached to argument types is that it would allow "higher-level" templating, e.g.,
```
@omnisci("int32(inputT, inputT, RowMultiplier, outputT | input_id=args<0>, outputT | input_id=args<1>)",
          inputT = ['Column<T>'], outputT = ['OutputColumn<T>'], T=['TextEncodedDict']) ...
```
compared to
```
@omnisci("int32(inputT, inputT, RowMultiplier, OutputColumn<T: input_id=args<0>>, OutputColumn<T: input_id=args<1>>", inputT = ['Column<T>'], T = ['TextEncodedDict']) ...
```
where output column types must be explicit when using `input_id` after the item type.